scripts/install.sh

```diff
@@ -3,14 +3,14 @@
 home_dir="gnu-koryavov"
 scipts_dir="scripts"
 gnukoryavov_name=gnu-koryavov
-dependencies=(curl egrep sed wget)
+main_config=./../configs/editors
+source ./../configs/dependencies
```

Comment:
Maybe it would be better to use only 2 config files (e.g. `config.conf` and `install.conf`)? I noticed we have only 2 config usage scenarios, so we can add something like `install.conf` with the following contents instead of 3 config files:
```shell
dependencies=(curl egrep sed wget)
editors=(okular atril evince xreader zathura)
```

(and use `$install_config`)